experimental/javascript_nodejs/qnamaker-activelearning-bot/README.md

```diff
@@ -0,0 +1,122 @@
+# QnA Maker sample
+Bot Framework v4 QnA Maker bot sample
+
+This bot has been created using [Microsoft Bot Framework][1], it shows how to create a bot that uses the [QnA Maker Cognitive AI][19] service.
+
+The [QnA Maker Service][19] enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured documents or editorial content in minutes. In this sample, we demonstrate how to use the QnA Maker service to answer questions based on a FAQ text file used as input.
+
+## Prerequisites
+This samples requires prerequisites in order to run.
+- [Required Prerequisites][41]
+
+# To try this sample
+- Clone the repository
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
+- In a terminal, navigate to `experimental/javascript_nodejs/qnamaker-activelearning-botsamples`
```

Comment:
I think the path you provide here isn't quite correct.  shouldn't it be `experimental/javascript_nodejs/qnamaker-activelearning-bot`